.github/workflows/release.yaml

```diff
@@ -20,13 +38,30 @@ jobs:
         with:
           java-version: 11
 
-      - name: Build wrapper with Maven
-        id: build_wrapper
-        run: mvn -B clean package --file ./wrapper/pom.xml
-
       - name: Get release version
         id: get_version
-        run: echo ::set-output name=VERSION::${GITHUB_REF:11}
+        run: echo ::set-output name=VERSION::${GITHUB_REF:12}
+
+      - name: Create new devlopment version
+        id: bump_version
+        uses: christian-draeger/increment-semantic-version@1.0.2
+        with:
+          current-version: ${{ steps.get_version.outputs.VERSION }}
+          version-fragment: 'feature'
+
+      - name: Configure git
+        id: configure_git
+        run: |
+          git config --global user.name 'Lambda wrapper CI'
+          git config --global user.email 'lambda-wrapper-ci@users.noreply.github.com'
+
+      - name: Prepare release
+        id: prepare_release
+        run: ./pre-release.sh ${{ steps.get_version.outputs.VERSION }}
```

Comment:
WDYT about hiding those `(pre|post)-release.sh` files in the `.github` directory? They're not supposed to be used by any human being directly, they're tightly related to the release workflow.

Reply:
Done.